_layouts/home.html

```diff
@@ -4,18 +4,21 @@
 <div class="hero">
   <div class="fixed-width">
     <a class="site-title" href="{{ site.baseurl }}/">
-      <img class="logo" src="{{ '/images/logo-text-white.png' | prepend: site.baseurl }}" alt="Student Robotics"/>
+      <img class="logo" src="{{ '/images/logo-text-white.png' | prepend: site.baseurl }}" alt="Student Robotics" />
     </a>
     {% include navigation.html class="site-navigation home-navigation" %}
   </div>
   <div class="hero-container hero-homepage">
     <div class="fixed-width">
       <div class="hero-text">
         <h1>An annual robotics competition for young people aged 16-18</h1>
-        <a class="button button-hero" href="{{ '/about' | prepend: site.baseurl }}">Learn more</a>
+        <a class="button hero-homepage-button" href="{{ '/compete' | prepend: site.baseurl }}">Compete</a>
```

Comment:
Question: There's a lot of duplication between these CTAs and the navbar - is that intentional?